README.md

```diff
@@ -161,10 +162,11 @@ Set up cron to schedule three jobs - crawling for new data, emailing alerts to u
 
 ```bash
 $ crontab -e
-0 14 * * *  cd /path_to_code/meirim/ && NODE_ENV='production' /usr/bin/node /path_to_code/meirim/bin/iplan >> /path_to_code/meirim/logs/combined.log 2>&1
-* * * * *  cd /path_to_code/meirim/ && NODE_ENV='production' /usr/bin/node /path_to_code/meirim/bin/send_emails >> /path_to_code/meirim/logs/combined.log 2>&1
-30 * * * *  cd /path_to_code/meirim/ && NODE_ENV='production' /usr/bin/node /path_to_code/meirim/bin/aggregate_views >> /path_to_code/meirim/logs/combined.log 2>&1
```

Comment:
@GalGend 
missing from current cron, is that on purpose?

Reply:
no, they're needed, can you please add?

Reply:
i returned aggregate_views, i hope u added to prod real crontab